app/index.html

```diff
@@ -71,98 +79,81 @@ <h1 class="brand-logo center">Cartograms 4 All</h1>
             </div>
           </div>
           <div class="card-action">
-            <a href="#">This is a link</a>
-            <a href="#">This is a link</a>
+    
           </div>
         </div>
       </div>
     </div>
 
-    <!-- Customize Modal Structure -->
-    <div id="modal" class="modal modal-fixed-footer">
-      <div class="modal-content">
-        <span class="close">&times;</span>
-        <h4>Custom CSV</h4>
-        <p>Input data as a comma separated list.</p>
-
-        <div class="row">
-          <div class="col s1">
-            <div class="cust_data">
-             
-            </div>
-          </div>
-          <div class="col s1">
-            <div class="cust_data">
-              
-              
-             
+      
+      
+      
+      
+  <!-- Share Modal -->     
+  <div id="share_modal" class="modal">
+    <div class="modal-content">
+      <h5>Share</h5> 
+      <div class="center-align">
+           <a class="waves-effect waves-light btn-large" onclick="share_email()"><i class="material-icons left">email</i>e-Mail</a>
+           <a class="waves-effect waves-light btn-large" onclick="share_twitter()"><i class="material-icons left">send</i>Twitter</a>
+           <a class="waves-effect waves-light btn-large" id="download_svg"><i class="material-icons left">file_download</i>SVG</a>
+           <a class="waves-effect waves-light btn-large" onclick="download_png()"><i class="material-icons left">file_download</i>PNG</a>
+           <a class="materialize-textarea"> 
+      </div>
+               <!--
+              <textarea class="session-id-area" cols="20" id="copy_session_id" onclick="shareSessionID(this)" readonly></textarea>
+               -->
+            <div class="input-field col s12">
+               <input disabled value="Session ID" id="disabled" type="text" class="validate">
+               <label for="disabled"></label>
             </div>
-          </div>
-        </div>
+           </a>
+    </div>
+    <div class="modal-footer">
+      <a href="#!" class="modal-action modal-close waves-effect waves-green btn-flat">Close</a>
+    </div>
+  </div>
 
-      </div>
-      <div class="modal-footer">
-        <a href="#!" class="modal-action modal-close waves-effect waves-green btn-flat " onclick="run()">Submit</a>
-      </div>
+  <!-- Backup Session Info modal -->      
+  <div id="session_modal" class="modal">
+    <div class="modal-content">
+      <h5>Backup</h5> 
+        <div class="center-align">
+           <a class="waves-effect waves-light btn-large" onclick="saveSession()">Save your session</a>
+           <a class="waves-effect waves-light btn-large" onclick="loadSession()">Load your session</a>
```

Comment:
More concise if its Save/Load since this is a tab dedicated to sessions, we don't really need to repeat it